### PR TITLE
Add complaints page

### PR DESCRIPTION
**Pull request by raamSoftwire**
## Context

We need a complaints procedure. This is a bare-bones MVP which adds a new page which links to a google form.

## Changes proposed in this pull request

Add new links to the candidate and provendor footers
Add a new page with some guidance and a link to the google form

image

image

## Guidance to review

[Link to prototype](https://apply-beta-prototype.herokuapp.com/complaints)
The card says that the URL should be top level `https://www.apply-for-teacher-training.service.gov.uk/make-a-complaint` , but at the moment it is namespaced `/candidate/make-a-complaint` similar to the existing content pages.

## Link to Trello card

https://trello.com/c/rtIl0Cuo/2670-dev-complaints-page-mvp

## Things to check

- [x] This code does not rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

File: app/views/content/complaints.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+The Department for Education is committed to providing a high level of customer service, and takes complaints seriously. Letting us know if something is wrong means we can fix it and improve the service for everyone.
+
+##Giving feedback
```
Comment (paulrobertlloyd) [inline app/views/content/complaints.md:3]: Just me being nit-picky…

```suggestion
## Giving feedback
```

File: app/views/content/complaints.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+If you have feedback about the service, or want to give suggestions for improvement, you can email us as [becomingateacher@digital.education.gov.uk](mailto:becomingateacher@digital.education.gov.uk).
+
+##How to make a complaint
```
Comment (paulrobertlloyd) [inline app/views/content/complaints.md:7]:
```suggestion
## How to make a complaint
```

File: app/views/content/complaints.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+To make a complaint about the Apply for teacher training service, please [complete our complaints form](https://docs.google.com/forms/d/e/1FAIpQLSdkaYmy03JP0s9gxfEOkBMvMwNuU7IYO_MLwFONg5GdVFBuFg/viewform).
+
+##How we’ll respond
```
Comment (paulrobertlloyd) [inline app/views/content/complaints.md:10]:
```suggestion
## How we’ll respond
```

File: app/views/content/complaints.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+The Department for Education is committed to providing a high level of customer service, and takes complaints seriously. Letting us know if something is wrong means we can fix it and improve the service for everyone.
+
+##Giving feedback
+
+If you have feedback about the service, or want to give suggestions for improvement, you can email us as [becomingateacher@digital.education.gov.uk](mailto:becomingateacher@digital.education.gov.uk).
```
Comment (paulrobertlloyd) [inline app/views/content/complaints.md:5]:
```suggestion
If you have feedback about the service, or want to give suggestions for improvement, you can email us as <becomingateacher@digital.education.gov.uk>.
```

File: config/routes.rb [hunk 5]
```diff
@@ -19,6 +19,7 @@
 
     get '/accessibility', to: 'content#accessibility'
     get '/cookies', to: 'content#cookies_candidate', as: :cookies
+    get '/make-a-complaint', to: 'content#complaints', as: :complaints
```
Comment (paulrobertlloyd) [inline config/routes.rb:22]: Is there any reason why we can’t make this a top-level path? It’s not a candidate-specific page, but applies to the whole service, Apply and Manage.

**Review by paulrobertlloyd — DISMISSED** [review]

File: config/routes.rb (hunk 5 above)
Comment (raamSoftwire) [reply config/routes.rb:22]: All of our routes so far are namespaced (see the accessibility and cookies paths above), so I am keeping with that pattern.

But it is possible to make it top-level if we wanted.

File: app/views/content/complaints.md (hunk 4 above)
Comment (raamSoftwire) [reply app/views/content/complaints.md:5]: Neat, I didn't know you could do that. Cheers @paulrobertlloyd

**Review by davidgisbey — APPROVED** [review]
lgtm

**Pull request merged**